install.md

```diff
@@ -212,6 +212,65 @@ Building in a container is simpler, but more restrictive:
 $ make binary
 ```
 
+### Build and Install shell completion scripts
```

Comment:
Does this need an extra step? AFAICS `make install` would, as of this PR already, cause generation of the completion scripts, and then install the completion scripts, just as it installs man pages or config files without having to be specifically instructed.

Reply:
yes I think make install covers it but there can be cases where you would want to only generated the script.
Not everyone is using the make rules. Given that this removes the current script from the repo it would be good to have some doc on how to recreate it.

Reply:
`install.md` guides users through what we think is the best way to install Skopeo. It’s not a reference documentation for `Makefile` documenting every target and variable. (We don’t really have that anywhere… and I’m not sure we should, at least given the current shape of the `Makefile`: the targets and variables range from clearly end-user-targeted to clearly internal implementation details, with a lot, sadly, somewhere in the middle.)

> but there can be cases where you would want to only generated the script.

Like what? It seems to me that script and the binary should match, so if both are installed, it always makes sense to do both together. Is that not the case?